Dockerfile

```diff
@@ -20,7 +20,7 @@ FROM ${DISTRO} AS build
 
 WORKDIR /src
 
-RUN git clone https://github.com/nodejs/node.git .
+RUN git clone https://github.com/nodejs/node.git . --depth 1
```

Comment:
I wanted to do this as well, but then we can checkout the target commit as we have no depth.

Another option is to skip the commit validation table and just clone the commit or branch directly with no depth, which I didn't want to do in the original PR.